modules/insticatorBidAdapter.js

```diff
@@ -190,31 +197,96 @@ function buildDevice(bidRequest) {
   return device;
 }
 
+function _getCoppa() {
+  return config.getConfig('coppa') === true ? 1 : 0;
+}
+
+function _getGppConsent(bidderRequest) {
+  let gpp = deepAccess(bidderRequest, 'gppConsent.gppString')
+  let gppSid = deepAccess(bidderRequest, 'gppConsent.applicableSections')
+
+  if (!gpp || !gppSid) {
+    gpp = deepAccess(bidderRequest, 'ortb2.regs.gpp', '')
+    gppSid = deepAccess(bidderRequest, 'ortb2.regs.gpp_sid', [])
+  }
+  return { gpp, gppSid }
+}
+
+function _getUspConsent(bidderRequest) {
+  return (deepAccess(bidderRequest, 'uspConsent')) ? { uspConsent: bidderRequest.uspConsent } : false;
+}
+
 function buildRegs(bidderRequest) {
+  let regs = {
+    ext: {},
+  };
   if (bidderRequest.gdprConsent) {
```

Comment:
Check how rubicon adapter works, they pass consent string regardless of gdpr consent object is true or not. We should do that too since the consentString can be there for some Non-EU countries too. Please update the same for user object too

Reply:
Done! @rohanInsticator

Reply:
Actually, if you latest code of rubicon.
![Screenshot 2024-03-26 at 11 28 18 AM](https://github.com/Insticator/Prebid.js/assets/127132399/c6ac73bd-a605-4085-a080-80459880dc3b)

They do check for `gdprConsent` and have removed the `user.ext.consent` as `regs.ext.gdprConsentString` servers the same purpose.
@rohanInsticator

Reply:
User consent is there in rubicon, not sure what you were checking

utils.deepSetValue(data, 'user.ext.consent', bidderRequest.gdprConsent.consentString);

Reply:
@rohanInsticator you are checking our repository, it's master is not synced with Prebid's official master branch.

Reply:
I was looking at PrebidJS repo only in my intellij. I cannot pull the latest on master over there somehow. Even if rubicon has removed it, they must be appending it on the Server Side as IAB said it's good to have that field inside User Object too.